src/main/java/ro/tripleaunibuc/application/auth/web/AngularRouteController.java

```diff
@@ -0,0 +1,19 @@
+package ro.tripleaunibuc.application.auth.web;
```

Comment:
We should delete this class since we won't use it, it's leftover from an example using Angular which we don't have